hecuba_core/src/py_interface/UnitParser.h

```diff
@@ -163,5 +163,18 @@ class UuidParser : public UnitParser {
     virtual PyObject *c_to_py(const void *payload) const;
 };
 
+class TupleParser : public UnitParser {
+private:
+    ColumnMeta col_meta;
+public:
```

Comment:
The public section comes first, then the protected, and the last one must be the "private:" section. Please switch them :)

This is coding standard in C/C++ so future developers know which methods can access without reading all the definitions, because he can't use the private section